iron-fit-behavior.html

```diff
@@ -39,6 +39,7 @@
        * The element that will receive a `max-height`/`width`. By default it is the same as `this`,
        * but it can be set to a child element. This is useful, for example, for implementing a
        * scrolling region inside the element.
+       * @type {!Element}
```

Comment:
"!Element": non-null Element
"?Element": Element|null
"Element": Element|null
"Element=": Element|undefined|null
"!Element=": Element|undefined

I know it's all very confusing :-/